build-tools/makefile_components/base_container.mak

```diff
@@ -8,6 +8,9 @@ SANITIZED_DOCKER_REPO = $(subst /,_,$(DOCKER_REPO))
 
 DOTFILE_IMAGE = $(subst /,_,$(IMAGE))-$(VERSION)
 
+# In CI environments, use the plain Docker build progress to not overload the CI logs
```

Comment:
These changes to base_container.mak are irrelevant I think. No need to push a change though, because as long as we're not using this, it will come out in another PR.